src/views/CollectionReport.vue

```diff
@@ -76,8 +77,8 @@ import ReportListRow from '@/components/report-components/ReportListRow'
 import ReportSubCollection from '@/components/report-components/ReportSubCollection'
 import CollectionReportInfoCard from '@/components/cards/CollectionReportInfoCard'
 import CollectionSelector from '@/components/buttons/CollectionSelector'
-
 import { mapDetailsTableContent, mapCollectionsData, collectionReportInformation } from '@/utils/templateMapper'
+import { mapCollectionsData as mapCollectionsToJsonLD } from '@/utils/bioschemasMapper'
```

Comment:
Make this consistent with the other import. Try to make propertynames more specific. then you do not need to do this

e.g:
getCollectionBioschemaData